src/components/BookCard.jsx

```diff
@@ -5,7 +5,12 @@ import Style from './BookCard.module.css';
 import 'react-circular-progressbar/dist/styles.css';
 
 /* eslint-disable jsx-a11y/anchor-is-valid, react/prop-types,react/jsx-curly-brace-presence */
```

Comment:
- [ ] As pointed out above, As required by Microverse you are not meant to use any linter `disable rule` in your codebase, the linters are there to make sure your code follows the best practices. I'll suggest you remove the linters disable rule and fix the errors. :+1: :100: